src/main/resources/templates/probes/listing.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en" data-layout-decorate="~{layouts/main}">
+<head>
+    <title>Foo</title>
+</head>
+<body>
+
+<main data-layout-fragment="content">
+    <div class="probes-listing--table">
+        <div class="probes-listing--header">
+            <span>ID</span>
+            <span>Name</span>
+            <span>Description</span>
+        </div>
+        <div data-th-each="probe : ${probes}" class="probes-listing--row">
```

Comment:
This is how we're using the object from java here in this HTML file. We're iterating over `${probes}` which has the same as our `ModelAndView` object.

Reply:
it's from the model part of the `ModelAndView`, yes.

Reply:
Yes, we defined the name of the view in the constructor and added model data to `ModelAndView` via the `addObject()` method.